src/ffi/pkgconfig.nit

```diff
@@ -67,27 +67,7 @@ private class PkgconfigPhase
 			end
 		end
 
-		# check availability of pkg-config
```

Comment:
To keep concern together, instead of cutnpaste, why not keeping the code in `pkgconfig.nit` in a specific function; then calling this new function from `abstract_compiler.nit`?

Reply:
this specific function may also be called at different time: earlier on linux, latter when crosscomiling...

Reply:
The real difference between the two checks in Nit code (the one in nitc and the one in niti) is the way errors are raised. Can we replace the use of `NaiveInterpreter::fatal` by `ToolContext::error`? Or insert a `check_error` afterwards?

Reply:
I agree. factorizing the check with the one in niti is the way to go